src/More/Licenses.tsx

```diff
@@ -8,16 +8,26 @@ import {
 } from "react-native"
 import { useTranslation } from "react-i18next"
 import { SvgXml } from "react-native-svg"
-import { getApplicationName } from "react-native-device-info"
 import env from "react-native-config"
 
 import { GlobalText } from "../components/GlobalText"
 
+import { getApplicationName } from "../gaen/nativeModule"
 import { Colors, Spacing, Typography } from "../styles"
 import { Icons } from "../assets"
 
 const Licenses: FunctionComponent = () => {
   const { t } = useTranslation()
+  const [applicationName, setApplicationName] = useState("")
```

Comment:
Not needed for this PR but I wonder if it would be good to have a `useApplicationName` hook at some point, given that we'll be using the name across the app.